src/Select/SelectInput.js

```diff
@@ -23,6 +23,10 @@ class SelectInput extends React.Component {
       // Rerender with the resolve `displayNode` reference.
       this.forceUpdate();
     }
+
+    if (this.props.autoFocus) {
```

Comment:
Should't we add a code branch for the non native mode only?

Reply:
```javascript
if (this.props.autoFocus && !this.props.native) {
      this.displayNode.focus();
}
```
That would be fine?